hadoop-common-project/hadoop-common/src/site/markdown/HadoopDocker.md

```diff
@@ -0,0 +1,63 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+## Hadoop Docker
+
+### Official releases
+
+There are special branches for running specific hadoop version in docker.
+These branches, use the files from official hadoop releases for running hadoop inside the containers.
+
+E.g.
+* [docker-hadoop-runner-latest](https://github.com/apache/hadoop/tree/docker-hadoop-runner-latest)
+* [docker-hadoop-runner-jdk11](https://github.com/apache/hadoop/tree/docker-hadoop-runner-jdk11)
+* [docker-hadoop-runner-jdk8](https://github.com/apache/hadoop/tree/docker-hadoop-runner-jdk8)
+* [docker-hadoop-runner](https://github.com/apache/hadoop/tree/docker-hadoop-runner)
```

Comment:
nit: `docker-hadoop-runner*` images are not related to any specific Hadoop version.  They are just the base images for running any Hadoop version.